connector_importer/models/recordset.py

```diff
@@ -145,6 +145,12 @@ def set_shared(self, values, reset=False):
         """Update import report values."""
         self.ensure_one()
         self._set_serialized('shared_data', values, reset=reset)
+        # Without invalidating cache we will have a bug because of Serialized
+        # field in odoo. It uses json.loads on convert_to_cache, which leads
+        # to all of our int dict keys converted to strings. Except for the
+        # first value get, where we get not from cache yet. So in order to
+        # streamline this I invalidate cache right away and use string keys
+        self.invalidate_cache(('shared_data',))
```

Comment:
sorry, but I see `_set_serialized` which deals w/ serialized fields... right? then, the fix should stay there to fix the issue w/ ALL serialized fields. No? :)